test/framework/integration_framework/fake_peer/yac_network_notifier.cpp

```diff
@@ -11,12 +11,6 @@
 
 namespace integration_framework {
 
-  void YacNetworkNotifier::subscribe(
-      const std::shared_ptr<iroha::consensus::yac::NetworkImpl>
-          &yac_transport) {
-    yac_transport->subscribe(shared_from_this());
```

Comment:
Since the method `shared_from_this()` is no more used. Can we remove `std::enable_shared_from_this` inheritance from `iroha/test/framework/integration_framework/fake_peer/yac_network_notifier.hpp` ?